charts/simple-workflow/templates/workflow-pipeline.yaml

```diff
@@ -176,6 +193,16 @@ spec:
               {{- else }}
               value: status.sync.status == Synced, status.health.status == Healthy, status.sync.comparedTo.source.targetRevision == {{ $.Values.app.targetRevision }}
               {{- end }}
+       {{- range $jobName, $jobConfig := $config.jobs }}
+        - name: verify-job-synced-and-healthy-{{ $name }}
+          depends: verify-job-healthy-{{ $name }}.Succeeded
```

Comment:
Where is this dependency defined?

Reply:
actually, we ll wait for the app to be healthy before checking if the job is healthy. This should be true with any type of Hooks.  reverting my change that I thought was a copy past err.